scripts/autoscaling/aws/nodeup.py

```diff
@@ -575,6 +577,11 @@ def label_node(nodename, run_id, api, cluster_name, cluster_role, aws_region):
     pipe_log('Instance {} is assigned to RunID: {}\n-'.format(nodename, run_id))
 
 
+def instance_is_active(ec2, instance_id):
+    status = get_current_status(ec2, instance_id)
+    return status == 16 or status == 0
```

Comment:
Could you please extract numbers `16` and `0` to constants?

Reply:
done